### PR TITLE
HEY! LISTEN! Gets rid of some active turfs

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Replaces some roundstart active turfs with inactive turfs. 

## Why It's Good For The Game
HEY! LISTEN!

### CODE REVIEW

**Comment by TerraGS** [comment]
MVP right here

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 1]
```diff
@@ -3204,9 +3204,7 @@
 /area/ruin/space/has_grav/ancientstation/engi)
 "ie" = (
 /obj/structure/table,
-/obj/item/stack/cable_coil{
-	amount = 30
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: this isn't an active turf.

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm [hunk 2]
```diff
@@ -4568,9 +4566,7 @@
 /obj/item/electronics/airalarm,
 /obj/structure/closet/crate/engineering/electrical,
 /obj/item/electronics/tracker,
-/obj/item/stack/cable_coil{
-	amount = 30
```
Comment (ShizCalev) [inline _maps/RandomRuins/SpaceRuins/oldstation.dmm]: .

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm (hunk 1 above)
Comment (kriskog) [reply _maps/RandomRuins/SpaceRuins/oldstation.dmm]: Map merger doing stuff, you should find they are 30 by default hence the replacement.

File: _maps/RandomRuins/SpaceRuins/oldstation.dmm (hunk 2 above)
Comment (kriskog) [reply _maps/RandomRuins/SpaceRuins/oldstation.dmm]: Same

**Pull request merged**